File: corpus/2018q2/_p2q1_ok/_check/A;0162.py
Scanner entrada = new Scanner (System.in)
Random rand = new Random ()
int cont =0, aux=0, k
int MAX = 50
print("Preencher vetor: ")
for i=0 in range(0, MAX):
for i=0 in range(0, MAX):
    print(V[i]+ " ")
print("\n")
print("O elemento do indice 6 é " +V[6])
for i=0 in range(0, MAX):
    if V[i] > V[6]:
        cont = cont +1
print("Existem " +cont + " números maiores do que " +V[6]+ " nesse vetor." )
print("Digite um número inteiro: ")
k = int(input())
for i=0 in range(0, MAX):
    print("O número " +k + " ocorreu na posição " +i + ".")
    aux=aux +1
if aux==0:
    print("O número " +k + " não foi encontrado no vetor.")

File: corpus/2018q2/_p2q1_ok/_check/A;0162.java

import java.util.Scanner;
import java.util.Random;

public class Questao1 {
    public static void main(String[] args) {

        Scanner entrada = new Scanner (System.in);
        Random rand = new Random ();   
        int cont =0, aux=0, k;

        int MAX = 50;
        int V[] = new int [MAX];
        System.out.println("Preencher vetor: ");
        for (int i=0; i<MAX; i++) {
            V[i] = (int)(Math.random()*72-21);
        }
        for (int i=0; i<MAX; i++) {
           System.out.print(V[i]+ " ");
        }
        System.out.println("\n");
        System.out.println("O elemento do indice 6 é " +V[6]); 
        for (int i=0; i<MAX; i++) {
           if (V[i] > V[6]) {
               cont = cont+1;
           }
        }
                System.out.println("Existem " +cont+ " números maiores do que " +V[6]+ " nesse vetor." );

        System.out.println("Digite um número inteiro: ");
        k = entrada.nextInt();

        for (int i=0; i<MAX; i++) {
           if (V[i] == k) {
               System.out.println("O número " +k+ " ocorreu na posição " +i+ ".");
               aux=aux+1;
           }
        } 
        if (aux==0) {
               System.out.println("O número " +k+ " não foi encontrado no vetor.");
        }
    }

}   
